components/Content/Articles.jsx

```diff
@@ -46,7 +46,7 @@ const Articles = ({ limit, showSeeAll }) => {
           )}
         </div>
 
-        <div className="grid gap-8 lg:grid-cols-3">
+        <div className="grid gap-8 md:grid-cols-2 lg:grid-cols-3">
```

Comment:
can we do the same two columns in md size for videos? (videos are on top on the pic)
<img width="980" alt="image" src="https://github.com/user-attachments/assets/f3b49ae3-7e4e-46a2-b88a-dd2fe897fb70">